packages/workflow/src/Expression.ts

```diff
@@ -22,20 +22,17 @@ import type { Workflow } from './Workflow';
 import { extend, extendOptional } from './Extensions';
 import { extendedFunctions } from './Extensions/ExtendedFunctions';
 import { extendSyntax } from './Extensions/ExpressionExtension';
+import { IS_V1_RELEASE } from './Constants';
+import { isExpressionError, isFrontend, isSyntaxError, isTypeError } from './utils';
 
 // Set it to use double curly brackets instead of single ones
 tmpl.brackets.set('{{ }}');
 
 // Make sure that error get forwarded
 tmpl.tmpl.errorHandler = (error: Error) => {
-	if (error instanceof ExpressionError) {
-		if (error.context.failExecution) {
-			throw error;
-		}
-
-		if (typeof process === 'undefined' && error.clientOnly) {
-			throw error;
-		}
```

Comment:
Not quite sure why we have this preexisting duplication and also whether `clientOnly` (used only once) is needed or can be simplified, but I'd rather keep this PR focused on the v1 changes.